.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Elixir CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Elixir
+      uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
```

Comment:
This `erlef/setup-beam` is new for me. Is it better than the classic `actions/setup-elixir`? Also, any reason to use this specific ref? What about `erlef/setup-beam@1.9`?